pymc_marketing/plot.py

```diff
@@ -598,6 +604,12 @@ def plot_curve(
     non_grid_names : str | set[str]
         The names to exclude from the grid. HDI and samples both
         have defaults of hdi and chain, draw, respectively
+    n_samples : int, optional
+        Number of samples
+    hdi_probs : float | list[float], optional
+        HDI probabilities. Defaults to None or 95%
```

Comment:
this needs to be checked. Perhaps there is a difference between Arviz versions, but for me (on ArviZ 0.21.0), the default is 94%.